meta.nabl2.java/src/main/java/org/metaborg/meta/nabl2/solver/IPublicSolution.java

```diff
@@ -25,5 +28,7 @@
     Map<String, IVariantRelation.Immutable<ITerm>> relations();
 
     ISymbolicConstraints symbolic();
+    
+    IControlFlowGraph<CFGNode> controlFlowGraph();
```

Comment:
Is this always there, or can it be null? In the latter case, nark it as @Nullable, or, preferably, make it optional.

Reply:
It's always there, just might be empty